code/model/model.h

```diff
@@ -1224,7 +1224,10 @@ void model_set_thrust(int model_num = -1, mst_info *mst = NULL);
 //   distance from eye_pos to closest_point.  0 means eye_pos is 
 //   on or inside the bounding box.
 //   Also fills in outpnt with the actual closest point.
-float model_find_closest_point(vec3d *outpnt, int model_num, int submodel_num, matrix *orient, vec3d * pos, vec3d *eye_pos);
+float model_find_closest_point(vec3d *outpnt, int model_num, int submodel_num, const matrix *orient, const vec3d *pos, const vec3d *eye_pos);
+
+// Like the above, but finds the closest two points to each other.
+float model_find_closest_points(vec3d *outpnt1, int model_num1, int submodel_num1, const matrix *orient1, const vec3d *pos1, vec3d *outpnt2, int model_num2, int submodel_num2, const matrix *orient2, const vec3d *pos2);
```

Comment:
Differing two functions by only an `s` could lead to readability issues. Maybe use something like `model_find_closest_point_on_each`?

Reply:
I'm of the opposite opinion: the similar names make it clear that the functions are closely related.  I don't think it's likely that the functions will be confused since one of them has twice as many arguments.

Reply:
Well, I don't care enough about that to not approve :stuck_out_tongue: